apistructs/service.go

```diff
@@ -218,6 +218,38 @@ type ServicePort struct {
 
 const (
 	ServiceDiscoveryKindProxy = "PROXY"
+
+	// CSI Vendor
+	CSIVendorAlibaba = "AliCloud"
+	CSIVendorTencent = "TecentCloud"
+	CSIVendorHuawei  = "HuaweiCloud"
+	// 对应磁盘类型为 SSD 磁盘
+	VolumeTypeSSD = "SSD"
+	// 对应磁盘类型为 NAS (NFS)磁盘
+	VolumeTypeNAS = "NAS"
+	// 对应磁盘类型为 OSS 磁盘
+	VolumeTypeOSS = "OSS"
+	// 对应磁盘类型为 DICE NAS (DICE NFS)磁盘
+	VolumeTypeDiceNAS = "DICE-NAS"
+	// 对应磁盘类型为 DICE LOCAL (DICE LOCAL)磁盘
+	VolumeTypeDiceLOCAL = "DICE-LOCAL"
+	// 阿里云 SSD 云盘 storageclass 名称
+	AlibabaSSDSC = "alicloud-disk-ssd-on-erda"
+	// 阿里云 NAS 网盘的 storageclass 名称
+	AlibabaNASSC = "alicloud-nas-subpath-on-erda"
```

Comment:
The NAS requires two types of storage class.
- use for pipeline job, doesn't set quota.
- use for addon, need limit quota.

Reply:
support in future.